catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/utils/FormulaEditorWrapper.java

```diff
@@ -130,6 +132,16 @@ public void performClickOn(Matcher<View> matcher) {
 				.perform(click());
 	}
 
+	public void performLongClickOn(Matcher<View> matcher) {
```

Comment:
since theres only one usage for each of those two methods, and they are rather specific for those tests they should not be in this wrapper (nobody else is going to need those, and theres little added readability)